sdk/storage/storage-blob/src/PageBlobRangeResponse.ts

```diff
@@ -1,23 +1,25 @@
 import { HttpResponse } from "@azure/core-http";
-import * as Models from "./generated/src/models";
+import { PageBlobGetPageRangesHeaders, PageBlobGetPageRangesDiffHeaders } from "./generatedModels";
+import {
+  PageBlobGetPageRangesResponse as PageBlobGetPageRangesResponseModel,
+  PageBlobGetPageRangesDiffResponse as PageBlobGetPageRangesDiffResponseModel
+} from "./generated/src/models";
 import { Range } from "./Range";
 
 export interface PageList {
   pageRange?: Range[];
   clearRange?: Range[];
 }
 
-export interface PageBlobGetPageRangesResponse
-  extends PageList,
-    Models.PageBlobGetPageRangesHeaders {
+export interface PageBlobGetPageRangesResponse extends PageList, PageBlobGetPageRangesHeaders {
   /**
    * The underlying HTTP response.
    */
   _response: HttpResponse & {
     /**
      * The parsed HTTP response headers.
      */
-    parsedHeaders: Models.PageBlobGetPageRangesHeaders;
```

Comment:
PRs #5632 and #5567 conflicted a bit here so this change is needed to resolve the conflicts and make api-extractor work correctly